k8s/standalone/graphscope-dev.Dockerfile

```diff
@@ -0,0 +1,17 @@
+# the graphscope-dev image is based on graphscope-dev-base, and will install
```

Comment:
Why do we need these two images???? a little weird.

Reply:
In order to speed up the building of graphscope-dev, cause vineyard will be upgraded more frequently than the stuffs in the dev-base..., It could be removed, though.

Reply:
How about add a _ prefix in the internal (won't be released) dockerfiles